src/components/Releases/ReleasesList.css

```diff
@@ -0,0 +1,7 @@
+ul {
```

Comment:
This might be the shortest css file I've ever seen. Excellent organization with modules.